ui/src/components/index.ts

```diff
@@ -0,0 +1 @@
+export { PinPreview } from "./PinPreview";
```

Comment:
Might be prudent to add the Pin component here too since you made the index already.